apps/wallet/src/background/keyring/VaultStorage.ts

```diff
@@ -132,17 +133,22 @@ class VaultStorageClass {
      * NOTE: make sure you verify the password before calling this method
      * @param keypair The keypair to import
      * @param password The password to be used to store the vault. Make sure to verify that it's the correct password (of the current vault) and then call this function. It doesn't verify the password see {@link VaultStorage.verifyPassword}.
+     * @param existingAccounts All the current account of the wallet derived and imported
```

Comment:
```suggestion
     * @param existingAccounts The wallet's derived and imported accounts
```
super nit: sorry to be the grammar police haha, this might read better